WalletWasabi.Backend/Controllers/SoftwareController.cs

```diff
@@ -13,6 +15,15 @@ public class SoftwareController : Controller
 	{
 		private readonly VersionsResponse VersionsResponse = new VersionsResponse { ClientVersion = Constants.ClientVersion.ToString(3), BackendMajorVersion = Constants.BackendMajorVersion };
 
+		public Global Global { get; }
+
+		private RPCClient RpcClient => Global.RpcClient;
+
+		public SoftwareController(Global global)
```

Comment:
Don't work into `SoftwareController`. It doesn't have API versioning.

1. If you want this to be a very specific filters check request as it is right now, then work into `BlockchainController`. In this case the "status"-ish naming scheme is not good.
2. But I see from your naming schemes that your idea is to add more stuff to this API call later on, in this case work into `OffchainController`. Also in this case drop the "status" naming and go with something like "health check" instead.